update-extension-dependencies.sh

```diff
@@ -49,7 +54,9 @@ echo ''
 # get all "artifact-id" values from the generated json file
 # pipefail is switched off briefly so that a better error can be logged when nothing is found
 set +o pipefail
-ARTIFACT_IDS=$(cd "${PRG_PATH}" && grep '^    "artifact"' devtools/bom-descriptor-json/target/quarkus-bom-quarkus-platform-descriptor-*.json | grep -Eo 'quarkus-[a-z0-9-]+' | sort)
+# note: quarkus-amazon-common was removed from this repo without a relocation
+ARTIFACT_IDS=$(cd "${PRG_PATH}" && grep '^    "artifact"' devtools/bom-descriptor-json/target/quarkus-bom-quarkus-platform-descriptor-*.json \
+    | grep -Eo 'quarkus-[a-z0-9-]+' | grep -v quarkus-amazon-common | sort)
```

Comment:
Maybe a stupid idea but should we just exclude all the relocated artifacts here? That might be easier?

Reply:
`-Prelocations` further up already does that trick! It prevents those relocated modules from ending up in the descriptor.
But since we don't have a relocation for `quarkus-amazon-common` and the generation of the descriptor doesn't respect the workspace, we have to exclude that single artifact like this.

Ideally, generation would take the workspace into account and then we'd be good, without any specific filtering.

Reply:
And to answer:
> That might be easier?

No, IMO. The list would be much longer and as long we have a relocation (also for future modules to be moved out), we're good here.

Reply:
well, the list would be longer but you wouldn't have to enforce the build :).

Anyway, I'm good with this.

Reply:
> you wouldn't have to enforce the build

That's true! Still, I prefer less stuff in the script (which is only a band aid after all).